cookbooks/convection-box/doc/convection-box.md

```diff
@@ -168,28 +174,29 @@ the same.
 ##### Visualizing results.
 
 Having run the program, we now want to visualize the numerical results we got.
-can generate graphical output in formats understood by pretty much any
+ASPECT can generate graphical output in formats understood by pretty much any
 visualization program (see the parameters described in
-Section&nbsp;[\[parameters:Postprocess/Visualization\]][4]) but we will here
-follow the discussion in Section&nbsp;[\[sec:viz\]][5] and use the default VTU
+Section&nbsp;{ref}`parameters:Postprocess/Visualization`) but we will here
+follow the discussion in Section&nbsp;{ref}`sec:viz` and use the default VTU
```

Comment:
Please replace 'Section&nbsp;{ref}`sec:viz` with {ref}`sec:run-aspect:visualizing-results`

(This section isn't technically setup yet so that reference tag is brand new, but it will be merged soon)